src/Console/Shell.php

```diff
@@ -708,13 +698,8 @@ public function info($message = null, $newlines = 1, $level = Shell::NORMAL)
      */
     public function warn($message = null, $newlines = 1)
     {
-        if (is_array($message)) {
-            foreach ($message as $k => $v) {
-                $message[$k] = '<warning>' . $v . '</warning>';
-            }
-        } else {
-            $message = '<warning>' . $message . '</warning>';
-        }
+        $messageType = 'warning';
```

Comment:
why not directly using the string in the method? The assignment seems not necessary.